CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -307,10 +327,12 @@ function PlayerHospital:msgMultiReceptionDesks()
 end
 
 --! Give advice to the user about maintenance of plants.
-function PlayerHospital:msgPlant()
+--!param placed (bool) If a plant was placed
+function PlayerHospital:msgPlant(placed)
   local num_handyman = self:countStaffOfCategory("Handyman", 1)
+  local plants = placed and self:countPlants() or 1
 
-  if num_handyman == 0 then
+  if num_handyman == 0 and plants == 1 then
```

Comment:
Instead of this cunning `1` plant trick, maybe

``` lua
if self:countStaffOfCategory("Handyman", 1) > 0 then return end

local num_plants = self:countPlants()
if num_plants == 0 or (placed and num_plants > 1) then return end
self:giveAdvice({_A.staff_advice.need_handyman_plants})
```